lib/usd/ui/layerEditor/layerTreeItemDelegate.cpp

```diff
@@ -258,41 +257,47 @@ void LayerTreeItemDelegate::paint_drawOneAction(
     }
 }
 
-void LayerTreeItemDelegate::paint_ActionIcons(
-    QPainter*     painter,
-    QRectC        rect,
-    Item          item,
-    const QColor& highlightColor) const
+void LayerTreeItemDelegate::paint_ActionIcon(
+    QPainter*       painter,
+    QRectC          rect,
+    Item            item,
+    LayerActionType actionType,
+    const QColor&   highlightColor) const
 {
+    LayerActionInfo action;
+    item->getActionButton(actionType, action);
+    LayerMasks layerMask
+        = CreateLayerMask(item->isRootLayer(), item->isSublayer(), item->isSessionLayer());
```

Comment:
This will require more code. The reason for CreateLayerMask is to simplify the | join as we almost always would want all flags to be set. In fact, at a point I thought about making a GetLayerMask() as a layer item member. Perhaps that's the way to go if there's going to be more calls in the future.